node/src/actors/chain_manager/mod.rs

```diff
@@ -1115,6 +1138,13 @@ fn show_info_dr(data_request_pool: &DataRequestPool, block: &Block) {
             )
         });
 
+    if Some(block_hash) == mined_block {
+        log::info!(
+            "Congratulations! Your block {} was consolidated in chain",
+            block_hash
```

Comment:
The block hash is redundant because it will be printed in the log immediately below
```suggestion
```

Reply:
What do you think about printing the number of mined blocks so far? Something like "your node has mined {} blocks so far".

Reply:
Will that be like the 40th entry in `struct ChainManager`? :stuck_out_tongue:

I think it's cool as long as it's in a `struct Stats` or similar where we can eventually throw any further metrics that we want to track in the future.

Reply:
Having a struct Strats would help for the Prometheus instrumentation of metrics I'm studying.

Reply:
@witnet/devs yet this will be a problem with bincode because we can't add new fields.

Reply:
done!!